### ISSUE TITLE
UX Review of updates to VBA Facility content type

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#14888]
## User Story or Problem Statement
Review implementation in #14701 

### Design

[Figma file](https://www.figma.com/file/aKdT4weNNthDxb3r6MolEh/VBA-Content-Specs?type=design&node-id=0%3A1&mode=design&t=HMbWiEC8fQQTUEiu-1)

## Tasks
- [x] Review editorial interface components for usability and compliance with CMS design system
- [x] Review all text (help text, field labels, tool tips, error messages) for UX writing best practices
  - [x] "Prepare for your visit" help text should be rewritten to avoid phrase "location services"
- [x] Consider broader user flow(s) for the task(s) and make sure this logically integrates/references other Drupal entities
- [x] Consider order of centralized content sections [comment from 14899](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14899#issuecomment-1761959612)
   - "🥝 The order of the centralized content sections doesn't seem to follow the order they appear on the Facility content type which seems confusing to me"
 - [x] Consider Veteran-facing labels [from comment in 14528](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14528#issuecomment-1780018883)
- [x] Consult with Engineering to understand what technical limitations may prevent updating interface
- [x] Document any recommendations for improving the interface

## Acceptance Criteria (Sprint 98) - 8 points

- [x] Recommendations are documented [in Figma](https://www.figma.com/file/aKdT4weNNthDxb3r6MolEh/VBA-Content-Specs?type=design&node-id=0%3A1&mode=design&t=HMbWiEC8fQQTUEiu-1)

## Acceptance Criteria (Sprint 99) - 5 points

- [x] Implement feedback
- [x] Review with UX Lead (Amanda)
- [x] Review with Content Designer (Dave)
- [x] Review with Engineers
- [x] Review with Accessibility

## Acceptance Criteria (Sprint 100) - 5 points
- [x] Implement feedback
- [x] Review with Content Designer (Dave)
- [ ] Review with Engineers (may be done at UX sync)
- [x] Share mockup for review by UX Lead (Amanda) - 48h turnaround suggested

**Comment by davidmpickett** [comment]
While this is not fully blocked by #14700, it might make sense to wait for that since it will be updating this content type

**Comment by mmiddaugh** [comment]
@thejordanwood as you review, please look for opportunities to give the editors context for the purpose/intended use of the content and/or link to (coming soon) KB articles
One detail I noticed this morning -> the guidance to avoid duplication of COVID-19 status (referenced in the operating status area) is irrelevant and should not be displayed

**Comment by thejordanwood** [comment]
I've started my UX review in [the Figma file](https://www.figma.com/file/aKdT4weNNthDxb3r6MolEh/VBA-Content-Specs?type=design&node-id=500%3A154&mode=design&t=SpLKfoqJ8nUVMYM8-1) and am working on getting feedback. 

A couple of notes about the ACs:

1. Consider order of centralized content sections - This looks pretty close to the order we have them in on the FE now. I believe this is okay as is. 
2. Consider Veteran-facing labels - @davidmpickett I think I need more guidance on this. I've looked at [the comment on 14528](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14528#issuecomment-1780018883) and am unsure what needs to change on the VBA Facility content type.

**Comment by thejordanwood** [comment]
**End of sprint update:**

I started [a Slack thread](https://dsva.slack.com/archives/C0FQSS30V/p1701468972917869) for feedback. I can make any updates to this whenever I get to [the CMS collab cycle feedback ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15989), so I'm going to call this finished for now.

**Comment by jilladams** [comment]
@thejordanwood FYI we discussed this ticket in sprint planning -- some notes: 
* Amanda noted we need to make sure UX review tickets do get the PO or UX lead review before closing moving forward, so leaving this ticket open to track updates from Michelle's comment above. 
* The notes from Michelle are re: fields that appear conditionally. We can address those 2 notes here. 
  * The Figma will in the end need to display not just the default form state, but the conditionals for any additional conditional fields. That bigger piece could be handled in the existing stubbed ticket for final Figma updates post-all-Drupal-work. (Dave estimated ~2-3 for that work, but @thejordanwood could you confirm?)
    * Note: I looked through the VBA epic and don't see that bigger picture Figma update ticket (just #16068 ), so we may need to confirm its existence. 
* @xiongjaneg noting: we will need a ticket (eventually) for engineers to make the updates that will be finalized in the Figma

**Comment by thejordanwood** [comment]
The design has been updated in [the Figma file](https://www.figma.com/file/aKdT4weNNthDxb3r6MolEh/VBA-Content-Specs?type=design&node-id=208%3A714&mode=design&t=NCy5FgnMRYSVhWi8-1). 

- The fields used in the banner alert are shown on the VBA Banners page in the Figma file. This is on a different page because it was part of a previous UX review. 
- I show the expanded Prepare for your visit accordions section and had two suggested changes. 
- I show the expanded Local spotlights section and have no suggested changes. Field labels are consistent with other fields in the CMS.

**Comment by davidmpickett** [comment]
This is a big piece of **Should** feedback about the order of the fields on this form. Might not be necessary to update the Figma file to reflect this, but just pass this along to Drupal folk in their [follow up ticket ](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16327)

- **Suggestion** - The order of elements in the Drupal form should generally match the order they those elements appear on the front end to help editors build a mental model of the impact of their edits.
- **Evidence** - [Vet Center Editor research](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/cms/research/vet-center) found "Matching the the front-end sequence streamlines content findability."
- **Detailed order suggestion**
  - Banner alert
  - Benefit office overview
  - Location and contact information
    - Facility operating status
    - Facility photo
    - Gray box of uneditable fields
      - Move "benefit hours may vary" and "benefits hotline" into this box
  - Prepare for your visit
  - In the spotlight
  - Facility Services 
     - Consider adding an HTML field here that simply explains that Facility Services (Veteran Benefits, family member benefits etc.) are created separately (with link to relevant KB)
   - Can't find the field you're looking for?
   - Get updates
 - Other VA locations
   - Consider adding an HTML field here that states that "other nearby VA locations" is generated automatically.

**Comment by davidmpickett** [comment]
> 2. Consider Veteran-facing labels - @davidmpickett I think I need more guidance on this. I've looked at [the comment on 14528](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14528#issuecomment-1780018883) and am unsure what needs to change on the VBA Facility content type.

In between the time I made that comment and you picked up this ticket, I worked with our Drupal devs to address this during their implementation, so it was already done. Sorry for any confusion!

**Comment by jilladams** [comment]
For @aklausmeier 's context when she returns: 

1. From Jordan in Slack: https://dsva.slack.com/archives/C0FQSS30V/p1702588501447089?thread_ts=1701468972.917869&cid=C0FQSS30V.  
> I've made some updates to the VBA Facility content type [in the Figma file](https://www.figma.com/file/aKdT4weNNthDxb3r6MolEh/VBA-Content-Specs?type=design&node-id=500%3A154&mode=design&t=NCy5FgnMRYSVhWi8-1) and now show expanded field groups. Feel free to give this another look!

2. Dave has 2 proposals that need signoff: 
    1. [in comment above](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14888#issuecomment-1857056106): rearrange Drupal form to field order to match FE field order
    2. In Figma annotations: some suggestions for new show/hide behavior and other Drupal suggestions. We could get yes/no on several of those.
        * There are some Drupal items that need @omahane  / @swirtSJW  to verify if they're technically possible. Following up on this [in Slack](https://dsva.slack.com/archives/C0FQSS30V/p1702921876686069?thread_ts=1701468972.917869&cid=C0FQSS30V). 

(@davidmpickett lemme know if anything there seems wrong to you.)

**Comment by davidmpickett** [comment]
Given @swirtSJW's feedback about hiding certain fields based on role not being a good direction, here's an alternate suggestion for the big gray box of uneditable information in Locations and Contact Information:
- SHOULD - Group the fields based on whether they display on the front-end webpage or not
- CONSIDER - Using the existing "DB data" box for grouping all the non-Veteran facing fields together
   - Facility Locator API ID
   - Classification
   - Geolocation
   - Shared VHA location
   - Rename the box to something plain language for editors
- SHOULD - Add a text explanation about the non-Veteran facing fields

**Comment by aklausmeier** [comment]
@thejordanwood  While you are working on a revised mockup....I added comments in Figma file to Facility page

Also, just noting that @mmiddaugh recommended linking back to relevant and helpful KB articles when possible in this [comment above](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14888#issuecomment-1830255540).

**Comment by jilladams** [comment]
@thejordanwood @xiongjaneg FYI chatted with Amanda and updated ACs here based on that convo -- please update if points don't seem right for what's left to do. Also assumed you'd need review from all the same folks - not sure if that's true.

**Comment by thejordanwood** [comment]
@aklausmeier @davidmpickett @swirtSJW @omahane I've created a new mockup [in the Figma file](https://www.figma.com/file/aKdT4weNNthDxb3r6MolEh/VBA-Content-Specs?type=design&node-id=500%3A154&mode=design&t=dmCcHvMMDWXtHGqT-1) to show Dave's order suggestions and other updates based on the feedback that's been provided. This is ready for review again.

**Comment by aklausmeier** [comment]
@thejordanwood FE and Drupal designs looks good, marking UX review complete

**Comment by jilladams** [comment]
For end of sprint: Just need to verify if there are any questions left for or from Dave / engineers. If not, can close

**Comment by davidmpickett** [comment]
Adding my face to account for reviewing effort

**Comment by davidmpickett** [comment]
✅ All my requested edits from the last round were implemented. I left a couple comments about help text, etc, but those are not blockers to closing this ticket

**Comment by davidmpickett** [comment]
@xiongjaneg @jilladams  Engineers have reviewing this Figma file as an AC in [their follow-up ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16327), so I don't see any reason to keep this one open.
